.github/workflows/upsert-scala-steward-dependencies-branch.yml

```diff
@@ -0,0 +1,22 @@
+name: Upsert `scala-steward-dependencies` branch
+
+on:
+  push:
+    branches: master
+jobs:
+  upsert-develop-branch:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    name: Rebase `scala-steward-dependencies` branch to latest `origin/master`
+    steps:
+      - name: Checkout scala-steward-dependencies branch
+        uses: actions/checkout@v2
+        with:
+          ref: scala-steward-dependencies
+          fetch-depth: 0
+      - name: Rebase `scala-steward-dependencies` branch to latest `origin/master`
+        run: |
+            git config user.email digitalpreservation@nationalarchives.gov.uk
```

Comment:
Are both the user email and user name needed?

Reply:
Tested this again on my local repo. Seems you don't need either of them.